bootstrap/bungeecord/src/main/java/org/geysermc/geyser/platform/bungeecord/GeyserBungeePingPassthrough.java

```diff
@@ -104,7 +105,7 @@ public int getVersion() {
         }
 
         @Override
-        public InetSocketAddress getVirtualHost() {
+        public @Nullable InetSocketAddress getVirtualHost() {
             return null;
         }
```

Comment:
was this actually giving warnings?

Reply:
Not build warnings, but i've got intelliJ set up in a way to flag null returns - can revert it; but for consistency, i'd mark it nullable if returns null